### PR TITLE
Little developing on your code.

### PR DESCRIPTION
**Pull request by sadrasabouri**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,15 @@ A tool to enumerate, list and monitor USB devices connected to a Linux host.
 
 The tool will print out a list of connected devices with USB path, vendor, serial and connection speed. Following that it will start monitoring for udev events such as connect or disconnect and print them with a timestamp. Uses `libudev`.
 
+Requirement
```
Comment (tenox7) [inline README.md:7]: This section is too long. Please make it much shorter. For example `requires libudev-dev`. That should be enough.

File: README.md [hunk 2]
```diff
@@ -19,4 +28,4 @@ Apache 2.0
 
 Disclaimer
 ----------
-This is not an official Google product.
\ No newline at end of file
+This is not an official Google product.
```
Comment (tenox7) [inline README.md:26]: Whats changed here?

File: usbtool.c [hunk 3]
```diff
@@ -27,68 +27,87 @@
 
 enum { INFO, WARNING, ERROR };
 
-void logmsg(int s, char *msg, ...) {
+int isopt(int argc, char **argv, const char *opt){
```
Comment (tenox7) [inline usbtool.c]: add space between `){`

File: usbtool.c [hunk 4]
```diff
@@ -27,68 +27,87 @@
...
+
```
Comment (tenox7) [inline usbtool.c]: Remove empty lines in the function

File: usbtool.c [hunk 5]
```diff
@@ -27,68 +27,87 @@
...
+    for(int i = 1; i < argc; ++i){
```
Comment (tenox7) [inline usbtool.c]: you can probably omit { } in for to make code shorter

File: usbtool.c [hunk 6]
```diff
@@ -27,68 +27,87 @@
...
+
+        if(strcmp(argv[i], opt) == 0)
```
Comment (tenox7) [inline usbtool.c]: you could move this condition in to the for statement, something like:
```
for(int i = 1; i < argc && strcmp(argv[i], opt) == 0; ++i)
   return 1;
```

File: usbtool.c [hunk 7]
```diff
@@ -27,68 +27,87 @@
...
     struct udev_device *dev;
     struct udev_list_entry *lst;
-    struct udev_list_entry *entr=NULL;
+    struct udev_list_entry *entr = NULL;
     const char *path, *usbpath, *vendor, *serial, *speed, *action;
     int fd,ret;
     fd_set fds;
 
-    udev=udev_new();
+    if(isopt(argc, argv, "--help")){
```
Comment (tenox7) [inline usbtool.c]: what about `-h` I typically use `-h` instead of `--help`

File: Makefile [hunk 8]
```diff
@@ -1,5 +1,5 @@
 all:
-	gcc -Wall -O3 -o usbtool usbtool.c -ludev
+	$(CC) -Wall -O3 -o usbtool usbtool.c -ludev
```
Comment (tenox7) [inline Makefile]: When updating Makefile perhaps refactor it to a more proper format. For example https://github.com/tenox7/ttyplot/blob/master/Makefile

**Review by tenox7 — CHANGES_REQUESTED** [review]
Thanks for updates! Please review my remarks.

File: README.md (hunk 1 above)
Comment (sadrasabouri) [reply README.md:7]: Yes!
You're right, i'll fix it.

File: README.md (hunk 2 above)
Comment (sadrasabouri) [reply README.md:26]: I think it most be a new line after this line.

File: usbtool.c (hunk 3 above)
Comment (sadrasabouri) [reply usbtool.c]: Allright.

File: usbtool.c (hunk 4 above)
Comment (sadrasabouri) [reply usbtool.c]: OK.

File: usbtool.c (hunk 5 above)
Comment (sadrasabouri) [reply usbtool.c]: Yes! :D

File: usbtool.c (hunk 6 above)
Comment (sadrasabouri) [reply usbtool.c]: Nice! :+1:

File: usbtool.c (hunk 7 above)
Comment (sadrasabouri) [reply usbtool.c]: Ok I'll add -h too.

Comment (sadrasabouri) [reply usbtool.c]: Ok I'll add -h too.

File: Makefile (hunk 8 above)
Comment (sadrasabouri) [reply Makefile]: Yes. You're right.

**Comment by sadrasabouri** [comment]
@tenox7 I've made some changes and pull request.
Please see the changes and tell me if any problem is there yet.

**Review by tenox7 — APPROVED** [review]

**Pull request merged**